src/gwproto/types/fibaro_smart_implant_component_gt.py

```diff
@@ -0,0 +1,86 @@
+import json
+import typing
+from typing import Any
+
+from gwproto.data_classes.component import Component
+from gwproto.data_classes.components.fibaro_smart_implant_component import (
+    FibaroSmartImplantComponent,
+)
+from gwproto.types import ComponentGt
+
+
+class FibaroSmartImplantComponentGt(ComponentGt):
+    ZWaveDSK: str = ""
```

Comment:
What is the ZWaveDSK?

Reply:
Actually I don't know. I just read it and assumed it was the ZWave unique id.